lib/util.js

```diff
@@ -2998,3 +2998,26 @@ util.estimateCores = function(options, callback) {
     }, 0);
   }
 };
+
+util.compareDN = function(l, r) {
+  var rval = false;
+
+  // compare hashes if present
+  if(l.hash && r.hash) {
+    rval = (l.hash === r.hash);
+  } else if(l.attributes.length === r.attributes.length) {
+    // all attributes are the same so issuer matches subject
+    rval = true;
+    var iattr, sattr;
```

Comment:
Should switch from this "issuer" "subject" naming to match the "l" "r" naming.

Reply:
Now ```dn1attr``` and ```dn2attr```